packages/vike-react-styled-jsx/README.md

```diff
@@ -85,31 +85,26 @@ const styledJsx = {
 }
 ```
 
-You can remove the styled-jsx SSR integration from [some of your pages](https://vike.dev/config#inheritance):
+You can remove the `vike-react-styled-jsx` integration from [some of your pages](https://vike.dev/config#inheritance):
 
 ```js
 // pages/about/+styledJsx.js
 
 export const styledJsx = null
 ```
 
-## What it does ?
-
-The `vite-react-styled-jsx` extension provides automatic integration of `styled-jsx`'s SSR capabilities with `vike-react`, making it easy to use `styled-jsx` for server-side rendering in your `vike-react` projects. What it does is:
-
-1. It creates the registry in [onBeforeRenderHtml()](https://vike.dev/onBeforeRenderHtml) by calling the `styled-jsx`'s `createStyleRegistry()` function and stores the resulting registry in `pageContext.styledJsx.registry`.
-   
-2. It wraps your app with `styled-jsx`'s `<StyleRegistry>` component and passes the registry to it via the [+Wrapper](https://vike.dev/Wrapper) component.
-
-3. It accesses the styles by calling `styled-jsx`'s `registry.styles()` and adds them to the `<head>` tags using [useConfig()](https://vike.dev/useConfig) in [onAfterRenderHtml()](https://vike.dev/onAfterRenderHtml).
-
 <br/>
 
 For full customization consider [ejecting](https://vike.dev/eject).
 
 > [!NOTE]
 > Consider making a [Pull Request before ejecting](https://vike.dev/eject#when-to-eject).
 
+## What it does ?
+
+The `vike-react-styled-jsx` extension automatically integrates `styled-jsx`'s SSR capabilities, preventing FOUC and scoping style rendering during each SSR pass to ensure concurrency safety. This allows the generated styles to be compared when the client loads, avoiding duplicate styles.  
+To learn more, have a look at the source code which is small. For full control [consider ejecting](https://github.com/vikejs/vike-react/pull/158#ejecting).
```

Comment:
I'm not great at making sentences 😅 , but anyway, what do you think?

Reply:
I used words from the styled-jsx [README](https://github.com/vercel/styled-jsx#server-side-rendering), except for 'preventing FOUC.'

Reply:
Let's first focus on content. Let's mention that it inject the styles generated upon SSR, in order to avoid FOUC.

> to ensure concurrency safety

This sounds (and is?) gibberish. Let's avoid this at all cost. Intervention += 3.

> This allows the generated styles to be compared when the client loads, avoiding duplicate styles.  

I think this is also inaccurate? Elaborate if it isn't. Please avoid this.

Reply:
Ah, indeed I'm reading `to keep concurrent-safe` from their docs. Weird.

Let's still avoid mentioning things we aren't sure what it means.

We can add a "Learn More at ..." if we want.

What we know for sure is that it avoids FOUC, and it's the classical FOUC integration. Maybe we can also link to https://vike.dev/css-in-js. Let's also popoulate `## See also`.

Reply:
Actually, I think I know what they mean. It's quite misleading and mentioning it doesn't add any value. Let's stick to the fundamentals we know about.

Reply:
Okay, I've updated the wording. Please take a look again 🙏